.gitignore

```diff
@@ -49,3 +49,4 @@ _preinstall
 
 # Debug files
 *.dSYM/
+.DS_Store
```

Comment:
can you please rm this, seems os specific I don't think it should be in the gitignore

Reply:
All right!